app/templates/index.html

```diff
@@ -21,14 +21,21 @@
         <script type="text/javascript" src="/static/js/controller.js"></script>
         <link rel="search" href="/opensearch.xml" type="application/opensearchdescription+xml" title="Whoogle Search">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="/static/css/{{ 'search-dark' if config.dark else 'search' }}.css">
         <link rel="stylesheet" href="/static/css/main.css">
-        {% if config.dark %}
-            <link rel="stylesheet" href="/static/css/dark-theme.css"/>
+        {% if config['theme'] %}
+            <link rel="stylesheet" href="/static/css/themes/{{config['theme']}}-search.css" />
+        {% else %}
+            <link rel="stylesheet" href="/static/css/{{ 'dark-' if config['dark'] else '' }}theme.css" />
```

Comment:
Seems like this could just be simplified to the code below, and remove the if/else block on lines 30-34.

```suggestion
        {% if config['theme'] %}
            <link rel="stylesheet" href="/static/css/themes/{{config['theme']}}.css" />
            <link rel="stylesheet" href="/static/css/themes/{{config['theme']}}-search.css" />
        {% else %}
            <link rel="stylesheet" href="/static/css/{{ 'dark-' if config['dark'] else '' }}theme.css" />
```